netlify.toml

```diff
@@ -765,12 +765,13 @@
     to = "/blog/building-the-future-of-game-analytics-pureskill"
 
 
-# Added: 2021-09-27
+# Added: 2021-9-28
```

Comment:
Pretty sure these are wrong i.e. https://posthog.com/handbook/engineering/production-access is the correct live URL.